core/services/keystore/csa_test.go

```diff
@@ -81,4 +82,35 @@ func Test_CSAKeyStore_E2E(t *testing.T) {
 		_, err = ks.Get(newKey.ID())
 		require.Error(t, err)
 	})
+
+	t.Run("adds an externally created key / ensures it already exists", func(t *testing.T) {
+		defer reset()
+
+		newKey, err := csakey.NewV2()
+		require.NoError(t, err)
```

Comment:
nit: require should probably be assert, since if this fails, we want the test to stop.

Reply:
Same for the require 2 lines below